config/config.dist.php

```diff
@@ -1,10 +1,13 @@
 <?php
+
+namespace Imbo\MetadataSearch;
+
 return [
     'eventListeners' => [
         'metadata' => [
             'listener' => 'Imbo\MetadataSearch\EventListener\MetadataOperations',
```

Comment:
Shouldn't we be consistant here? You removed the Imbo\MetadataSearch prefix from params, but not from the listener-name - is this intentional?